manim/scene/scene.py

```diff
@@ -348,18 +348,33 @@ def update_self(self, dt):
         for func in self.updaters:
             func(dt)
 
-    def should_update_mobjects(self):
+    def should_update_mobjects(self) -> bool:
         """
-        Returns True if any mobject in Scene is being updated
-        or if the scene has always_update_mobjects set to true.
+        Returns True if the mobjects of this scene should be updated.
 
-        Returns
-        -------
-            bool
+        In particular, this checks whether
+
+        - the :attr:`always_update_mobjects` attribute of :class:`.Scene`
+          is set to ``True``,
+        - the :class:`.Scene` itself has time-based updaters attached,
```

Comment:
Not sure if it's in the scope of this PR, but I think it should check whether any updaters are attached, not just time-based updaters.

Reply:
Wouldn't be hard to change at all, sure! Can change it tomorrow morning.

My first thought was that I'm not sure whether this is really necessary and has a practical use case, but I don't think it'd really hurt performance or semantics to check for any present updaters.

Reply:
In light of the recent discussions on Discord, I have developed a slight preference for keeping things as they are. If users really explicitly want to enforce frame-rate dependent behavior, then (a) their updater could get a dummy `dt` argument, or (b) they could use `self.wait(frozen_frame=False)` with this PR.

Reply:
SGTM